crates/lib3h/src/transport/memory_mock/ghost_transport_memory.rs

```diff
@@ -39,20 +44,55 @@ pub type GhostTransportMemoryEndpointContextParent = GhostContextEndpoint<
 
 #[allow(dead_code)]
 pub struct GhostTransportMemory {
+    machine_id: Address,
     network: String,
     endpoint_parent: Option<GhostTransportMemoryEndpoint>,
     endpoint_self: Detach<GhostTransportMemoryEndpointContext>,
     /// My peer uri on the network layer (not None after a bind)
     maybe_my_address: Option<Url>,
     /// Addresses of connections to remotes
     connections: HashSet<Url>,
+    last_discover: Option<Instant>,
```

Comment:
You might be able to get rid of the option and just set this to a sufficiently max or min value, but perhaps what you have here is clearer.